src/Symfony/Component/DependencyInjection/Loader/FileLoader.php

```diff
@@ -271,6 +270,10 @@ private function findClasses(string $namespace, string $pattern, array $excludeP
                 continue;
             }
 
+            if (!$info->isReadable()) {
```

Comment:
what about removing the call altogether? I don't think it's needed in practice

Reply:
great suggestion. I tried it locally and it worked for me.